aiohttp/web_response.py

```diff
@@ -160,12 +163,7 @@ def set_status(
             not self.prepared
         ), "Cannot change the response status code after the headers have been sent"
         self._status = int(status)
```

Comment:
Could probably also drop the int() call, atleast on master, given the type annotations.